README.md

```diff
@@ -47,6 +47,7 @@ No.
 | he_vm_mac_addr | null | MAC address of the engine vm network interface. |
 | he_domain_type | null | Storage domain type. available options: *nfs*, *iscsi*, *gluster*, *fc* |
 | he_storage_domain_addr | null | Storage domain IP/DNS address |
+| he_ansible_host_name | localhost | Host name to deploy on this host (not on the ansible controller) |
```

Comment:
I'd suggest something like:
`| he_ansible_host_name | localhost | hostname in use on the first HE host (not necessarily the Ansible controller one) |`

Reply:
Done !!